capnpc-ts missing dependency declaration on "debug"

In a fresh directory:

    $ mkdir node_modules
  
    $ npm install --save capnp-ts capnpc-ts
    capnp-ts@0.2.4 node_modules/capnp-ts
    ├── format@0.2.2
    ├── tslib@1.9.3
    ├── debug@2.6.9 (ms@2.0.0)
    └── utf8-encoding@0.1.2 (obtain-unicode@0.0.5)
    
    capnpc-ts@0.2.4 node_modules/capnpc-ts
    ├── hex2dec@1.0.1
    ├── bluebird@3.5.1
    ├── tslib@1.9.3
    ├── mkdirp@0.5.1 (minimist@0.0.8)
    ├── ramda@0.24.1
    └── typescript@2.9.2
  
    $ PATH=node_modules/.bin:$PATH capnp compile -o ts dashboard.capnp 
    module.js:549
        throw err;
        ^
    
    Error: Cannot find module 'debug'
        at Function.Module._resolveFilename (module.js:547:15)
        at Function.Module._load (module.js:474:25)
        at Module.require (module.js:596:17)
        at require (internal/module.js:11:18)
        at Object.<anonymous> (/home/kenton/src/capnp-ts-test/node_modules/capnpc-ts/lib/index.js:9:15)
        at Module._compile (module.js:652:30)
        at Object.Module._extensions..js (module.js:663:10)
        at Module.load (module.js:565:32)
        at tryModuleLoad (module.js:505:12)
        at Function.Module._load (module.js:497:3)
        at Module.require (module.js:596:17)
        at require (internal/module.js:11:18)
        at Object.<anonymous> (/home/kenton/src/capnp-ts-test/node_modules/capnpc-ts/bin/capnpc-ts.js:3:1)
        at Module._compile (module.js:652:30)
        at Object.Module._extensions..js (module.js:663:10)
        at Module.load (module.js:565:32)
    ts: plugin failed: exit code 1

It looks like `capnpc-ts` forgot to declare its dependency on `debug` (whereas `capnp-ts` does declare it correctly).

Work-around is to install `debug` manually:

    $ npm install --save debug
  
    $ PATH=node_modules/.bin:$PATH capnp compile -o ts dashboard.capnp 

This time it works.

--- a/packages/capnpc-ts/package.json
+++ b/packages/capnpc-ts/package.json
@@ -9,6 +9,7 @@
   "dependencies": {
     "bluebird": "^3.5.0",
     "capnp-ts": "^0.2.4",
+    "debug": "^3.1.0",
     "hex2dec": "^1.0.1",
     "mkdirp": "^0.5.1",
     "ramda": "^0.24.1",

